sip-tbd/sip-tbd-taproot_payments.md

```diff
@@ -0,0 +1,75 @@
+# Preamble
+
+SIP Number: [TBD]
+
+Title: Taproot Payments Integration
+
+Authors: Rena Shah ([rena@trustmachines.co](mailto:rena@trustmachines.co))
+
+Consideration: Governance, Technical
+
+Type: Consensus
+
+Status: Draft Proposal
+
+Created: 24 May 2022
+
+License: CC0-1.0
+
+Sign-off: Technical Committee
+
+# Abstract
+
+This SIP proposes a small but important change to enable mining payouts to Taproot scripts. Taproot scripts were enabled at block 709,632 on the Bitcoin Network on November 14, 2021. 
```

Comment:
Since this is proposing a hard fork, which is a very disruptive change, I don't think the text should not characterize it as "small."  It's going to be a lot of work to do this.

Reply:
Understood and I can update it accordingly!